assignment-01-cat/Cat.cpp

```diff
@@ -1,15 +1,69 @@
 #include "Cat.h"
+#include <cstring>
 
-Cat::Cat(const char* _name) {
+Cat::Cat(const char* _name)
+{
     // TODO: this->name에 동적할당을 하고 _name을 복사해야 합니다.
+    this->name = new char[strlen(_name)+1];
+    strcpy(name, _name);
     // TODO: this->age는 -1으로 초기화해야 합니다.
+    this->age = -1;
     // TODO: this->sex는 false로 초기화해야 합니다.
+    this->sex = false;
 }
 
-Cat::Cat(const char* _name, int _age) {
+Cat::Cat(const char* _name, int _age)
```

Comment:
```suggestion
Cat::Cat(const char* _name, int _age): Cat(_name, _age, false)
```
이런 식으로 다른 생성자를 호출해주면 매 생성자마다 문자열복사를 해주지 않아도 돼요!
코드 중복을 줄여줍시다.